thirdparty/kpvcrlib/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ include_directories(
     ${NANOSVG_INCLUDE_DIR}
     ${ZSTD_INCLUDE_DIR}
     ${CREGINE_INCLUDE_DIR}
+    ${SRELL_INCLUDE_DIR}
```

Comment:
Also move it above, the thirdparty _INCLUDE_DIR before the (mispelled) crengine own include dir - for consistency (dunno if it matters really :)